engine/src/flutter/lib/web_ui/lib/src/engine/js_interop/js_typed_data.dart

```diff
@@ -6,25 +6,24 @@ import 'dart:js_interop';
 
 // Some APIs we need on typed arrays that are not exposed by the dart sdk yet
 extension TypedArrayExtension on JSTypedArray {
-  external JSTypedArray slice(JSNumber start, JSNumber end);
-  external void set(JSTypedArray source, JSNumber start);
-  external JSNumber get length;
+  external JSTypedArray slice(int start, int end);
+  external void set(JSTypedArray source, int start);
+  external int get length;
 }
 
 // These are constructors on `Uint8Array` that we need that aren't exposed in
 // the dart sdk yet
 @JS('Uint8Array')
-@staticInterop
-class JSUint8Array1 {
+extension type JSUint8Array1._(JSObject _) implements JSObject {
```

Comment:
Do we have what we need here? https://api.dart.dev/stable/latest/dart-js_interop/JSUint8Array-extension-type.html

Reply:
Good call, removed the constructors and extension type.